### PR TITLE
Amls 5763 | New autocomplete for countries and currencies

### PR DESCRIPTION
**Pull request by slavvco**
As above.

## Related / Dependant PRs?

- none

## How Has This Been Tested?

- manually

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: public/amls.js [hunk 1]
```diff
@@ -302,96 +302,4 @@ $(function () {
               }
         }
   }
-
-  $('[data-gov-autocomplete]').each(function() {
```
Comment (ajchilts) [inline public/amls.js]: We need to ensure all additional entries listed here exist in the new control. 👍

File: public/autocomplete/accessible-autocomplete.min.js [hunk 2]
```diff
@@ -0,0 +1,1115 @@
+(function webpackUniversalModuleDefinition(e, t) {
```
Comment (ajchilts) [inline public/autocomplete/accessible-autocomplete.min.js]: Looks complex. Was this simply a grab and drop of the standard JS?

File: public/amls.js (hunk 1 above)
Comment (slavvco) [reply public/amls.js]: I will double check this, thanks

File: public/autocomplete/accessible-autocomplete.min.js (hunk 2 above)
Comment (slavvco) [reply public/autocomplete/accessible-autocomplete.min.js]: this is part of the component itself, I've just imported it :)

**Comment by slavvco** [comment]
Rebased. New PR: #1421

**Pull request closed without merging**